### DISCUSSION TITLE
[治安局] 特别作战开始

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#1885] [其他]
![mmexport1723539693652](https://github.com/user-attachments/assets/23cbdef5-4473-490a-8299-c4cfac527095)
8月13日17：00，刑侦特勤组朱鸢编号148接收到命令，已只身前往犯罪窝点…作战开始！

> 图源：pixiv：Ino

**Comment by wugan-bot156** [comment]
[图片的pixiv站来源](https://www.pixiv.net/artworks/12120759)
作者的x账号：[Ino /이노](https://x.com/kaa415309366715)

**Reply by ZhuYuan-148** [reply]
感谢这位市民！

**Comment by GanlinSama** [comment]
警光小心，注意安全（敬礼）

**Comment by HoshimiMiyabi120** [comment]
虽然以你的身手这些完全不是问题，但我还是要提醒一句，注意安全
